--- a/meson_post_install.py	2020-06-06 15:39:09.171812004 +0200
+++ b/meson_post_install.py	2020-06-06 15:39:30.305689083 +0200
@@ -26,7 +26,7 @@
         os.makedirs(dst)
         subprocess.check_call(['chmod', '700', dst])
         if getpass.getuser() == "root":
-            subprocess.check_call(['chown', polkitd_user, dst])
+            subprocess.check_call(['chown', '102', dst])
 
 # polkit-agent-helper-1 need to be setuid root because it's used to
 # authenticate not only the invoking user, but possibly also root
